src/backend/BSConfig.pm.template

```diff
@@ -155,8 +154,11 @@ our $relsync_pool = {
 #our $sign = '/usr/bin/sign';
 #Extend sign call with project name as argument "--project $NAME"
 #our $sign_project = 1;
+#Create a key by default for new projects, if top level project don't have one
+#our $forceprojectkeys = 1;
 #Global sign key 
 #our $keyfile = '/srv/obs/openSUSE-Build-Service.asc';
+#our $gpg_standard_key = "/etc/obs-default-gpg.asc";
```

Comment:
hm, can't we see $gpg_standard_key as legacy as well and drop it here? IMHO new instances shouldn't using this, the one instance wanting this is IMHO an exception ...